eng/AfterSolutionBuild.targets

```diff
@@ -5,10 +5,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <_PublishTarget Include="FullFramework" Framework="net461" RuntimeId="win-x64" />
```

Comment:
Does this `net461` target need to remain?

Reply:
Brought this back.